content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,71 @@
+---
+permalink: "rackspace-fabric-faq"
+audit_date: '2022-03-17'
+type: article
+title: "Rackspace Fabric FAQ"
+created_date: '2022-03-17'
+created_by: Asmita Nakwa
+last_modified_date: '2022-03-17'
+last_modified_by: Asmita
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+### General
+{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}
+
+If this is cloud account admin, they can review the following link (https://account.rackspace.com/event-history). It should help them understand any recent changes/access to their account.
+{{</accordion>}}
+
+{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
+
+The customer needs to add the firewall as a device and the **Add VPN User** template will be visible in common requests.
```

Comment:
I came up with the following for the answer:
`Rackspace is decommissioning the previous ticketing section and moving customers to the Fabric Ticketing experience which has already been used by multi-cloud users for some time. New features and customer feedback improvements will be implemented in the new Fabric Ticketing application and are currently in progress.`

Reply:
Added mentioned answer.